### ISSUE TITLE
Aframe AR demo incorrect scaling problem

### ISSUE THREAD

**Issue by Jason-LI-M** [jeromeetienne/AR.js#582]
**Describe the bug**
Aframe Demo, the cube have incorrect scaling or size. I need to resize the browser windows to fix it(i.e. video 1). If you go to Inspector view(i.e. video 2), you will see the cube have correct size and scaling. i guess this issue is related to canvas or canvas size.

**To Reproduce**
1. Go to 'https://jeromeetienne.github.io/AR.js/aframe/examples/basic.html'
2. Scan 'Hiro' marker

**Expected behavior**
The cube should have correct scaling / size

**Screenshots**
video 1: https://drive.google.com/open?id=1COxV-W9ulDr5ff1MiUfR1vcWT9b1_23f
video 2: https://drive.google.com/open?id=1Y_nt3mTA6WY_aHVg1XBZqG53nn_dtON-
image :
![aframe ar](https://user-images.githubusercontent.com/38548415/64007914-9f76f280-cb47-11e9-8b99-a3bb22d885d3.PNG)

**Desktop (please complete the following information):**
 - OS: Windows 7 64bit
 - Browser chrome
 - Version  chrome 76.0.3809.132

THX

**Comment by nicolocarpignoli** [comment]
maybe related to https://github.com/jeromeetienne/AR.js/issues/579

I switch back to 1.7.7 version on master, then I will take care of other issues about 1.7.8 when I will have more time

**Comment by nicolocarpignoli** [comment]
try now @Jason-LI-M

**Comment by POC-Maker** [comment]
I believe I've also been having the same issue on mobile devices (we've tried on both Android and iPhone). In my experience, the object had the correct scaling when the phone was held in landscape, but rotating it to portrait would stretch the object. I tried using 1.7.7, as per your suggestion earlier in the thread, and ran into the same issue. I'm also still having the issue with 1.7.8.

While googling, I also ran into a blog about ar.js development that seemed to be running into the same issue here: https://www.voucherify.io/blog/how-we-brought-our-product-mascot-to-life 
The quickfix in the blog seems to corroborate the idea that this issue is being affected by the canvas.

This also sounds similar to issue #60 so it could be related?
